bundle/language_en.properties

```diff
@@ -1527,6 +1527,8 @@ angal.visit.printthisdaysvisits1.btn
 angal.visit.printthisdaysvisits1.btn.key                                                               = 1
 angal.visit.printthisdaysvisits2.btn                                                                   = Print this day's visits (2)
 angal.visit.printthisdaysvisits2.btn.key                                                               = 2
+angal.visit.removevisit.btn                                                                            = Remove Visit
+angal.visit.removevisit.msg                                                                            = Remove Visit?
```

Comment:
Because this is used in a message dialog I would think the text should be more like `Remove the visit?`; WDYT

Reply:
alright! thanks!